polling_stations/settings/constants/councils.py

```diff
@@ -2,6 +2,9 @@
 
 YVM_LA_URL = "https://www.yourvotematters.co.uk/_design/nested-content/results-page2/search-voting-locations-by-districtcode?queries_distcode_query="  # noqa
```

Comment:
I think you can probably just ditch `YVM_LA_URL` here. In fact I probably I should have done that ages ago.